hermes-guide-days-website/src/app/footer/footer.component.css

```diff
@@ -0,0 +1,134 @@
+* {
```

Comment:
File should be scss, also * font-family at least should be moved styles.scss since the font is global